src/ansys/dyna/core/keywords/keyword_classes/auto/control_time_step.py

```diff
@@ -28,7 +28,7 @@ class ControlTimeStep(KeywordBase):
     """DYNA CONTROL_TIME_STEP keyword"""
 
     keyword = "CONTROL"
-    subkeyword = "TIMESTEP"
+    subkeyword = "TIME_STEP"
```

Comment:
Shouldn't this class name be modified too?

Reply:
It is modified. There will be two classes now for the sake of continuity even though CONTROL_TIME_STEP is not documented in the manual. It exists in the source of LS-PrePost.
ControlTimestep --> keyword = "CONTROL" , subkeyword = "TIMESTEP"
ControlTimeStep --> keyword = "CONTROL" , subkeyword = "TIME_STEP"

Reply:
Yep my bad again! Sorry @RaphaelHeiniger